packages/fxa-payments-server/src/components/PaymentConfirmation/index.tsx

```diff
@@ -172,7 +172,7 @@ export const PaymentConfirmation = ({
           </div>
         )}
 
-        <div className="footer" data-testid="footer">
+        <div className="payment-footer" data-testid="footer">
```

Comment:
I considered making a `PaymentFooter` component, but the footer only appears in these three templates (PaymentConfirmation, PaymentErrorView, and PaymentProcessing), so I was not sure if it's exactly necessary. Thoughts?

Reply:
Agreed. From our Slack discussion, if this is something that doesn't drastically change scope (this change would include SB and test updates across components as well) and/or you have bandwidth, this change can be made here, otherwise, a new ticket should be filed. I didn't realize how many components were affected; it is worth having in its own component and will prevent inconsistencies across the footer in future.

Reply:
Agreed.

From our Slack discussion, if this is something that doesn't drastically change scope (i.e., this change would also include SB and test updates across components) and/or you have bandwidth, this change can be made here, otherwise, a new ticket should be filed. I didn't realize how many components were affected; it is worth having in its own component and will prevent inconsistencies across the footer in future.

Reply:
I have filed a ticket ([FXA-6222](https://mozilla-hub.atlassian.net/browse/FXA-6222)) to create a PaymentFooter component, stories, and tests, as it is out of scope for FXA-5678. If I have the bandwidth during the current sprint, I will pull it in. Thanks!